service/controller/key/zz_generated.ami.go

```diff
@@ -1166,7 +1166,32 @@ var amiJSON = []byte(`{
     "us-east-2": "ami-033f192100c9980a1",
     "us-west-1": "ami-07a8a027c32ef1115",
     "us-west-2": "ami-08d7c3221e6f57e1f"
-   }
+   },
+  "3689.0.0": {
```

Comment:
this is a problem if you ask me. Why having an annotation to be able to use any flatcar version if we anyway need to add any new version into this file and make a new release?
I believe if we want the annotation in place we should get rid of this file and get AMI ID discovery somehow automated or at least prevent the need of a new release

Reply:
I agree, AMI ID discovery might be a good solution. 

It might be a bit more challenging in China since we're using private AMI's.

We might need to also touch the permissions for `aws-operator again, IIRC we don't have enough permissions right now to do the discovery.

Do you want me to add an issue for this idea?

Reply:
at least for tracking purposes, yes please

Reply:
ok will do

Reply:
Issue: https://github.com/giantswarm/giantswarm/issues/27990